common/lib/api-client/models.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
       date: '',
       date_from: '',
       date_to: '',
+      supplier_id: '',
```

Comment:
Can we delete this commit for cleanliness? It seems to be adding something that isn't available on the move?

Reply:
Zapped. It's been there for the past 4 months!